cryptol-remote-api/python/README.md

```diff
@@ -194,3 +194,8 @@ cryptol.connect(url="http://localhost:8080/", reset_server=True)
 
 will connect to a Cryptol server running at `http://localhost:8080/` and will
 guarantee any previous state on the server is cleared.
+
+
+## Supported Python Versions
+
+Currently `cryptol-remote-api supports` python `3.12` and is untested on other versions.
```

Comment:
There is one caveat to this statement: in the [`cryptol-remote-api` Dockerfile](https://github.com/GaloisInc/cryptol/blob/be32aaae7c0bc6ef6b0e2a93027882ce926ce53c/cryptol-remote-api/Dockerfile#L23) (not to be confused with `test.Dockerfile`, which you modified elsewhere in this PR), we use the version of Python provided by Ubuntu 22.04 (currently 3.10.12). Perhaps we should migrate this Dockerfile over to Python 3.12 for consistency's sake.

Given that we want to land this relatively soon (and that the fix will work without needing to change that Dockerfile), I propose that we open a separate issue to track this.

Reply:
#1661